README.md

```diff
@@ -2,65 +2,57 @@
 
 Decentralized wallet identification and authorization through message signing and signature validations.
 
+## Usage
+
+Stratis Open Auth Protocol can be used for the following purposes:
+
+* Personalising the data that a dApp stores about and displays to a user
+* Verifying that a user has access to the private key of a wallet address
+
 ## Benefits
 
 - Decentralized authentication
 - No third party access to sensitive user info
 
 ___
 
-## Wallet Implementation
+## Implementation
+
+### Stratis ID URI
+
+A Stratis ID URI can easily be parsed by a wallet. It consists of three required parts:
+
+- The scheme - `sid`
+- The callback - a protocol-relative URL which the wallet will send a HTTPS request
+- The UID - a unique identifier for a request, which is present within the query string of the callback
+
+**Example:** `sid:api.example.com/auth?uid=some-guid`
```

Comment:
Curious what is `sid:` in the example?

Reply:
This is the URI scheme, the wallet would parse it and be able to identify that the QR code is a Stratis ID URI.

Reply:
I see